tests/cargo-ui/list-harnesses/expected

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
I guess these kinds of tests are fine for now, but please put eliminating them on your todo list.

We should end-to-end test this, not look for specific output in the middle. What we eventually want is not these tests, but tests written against the IDE extension (that also runs as part of Kani's CI).

Reply:
will open an issue once this is closed. Agreed about testing the pipeline in parts.

Reply:
This is a command option in Kani, so it might not be a bad idea to keep it either.